lib/view_component/base.rb

```diff
@@ -249,6 +249,13 @@ def content_evaluated?
     #
     mattr_accessor :show_previews_source, instance_writer: false, default: false
 
+    # Path for component files
```

Comment:
👋🏻 mind running `rake docs:build` locally? Doing so will parse this file and update the API docs ❤️